doc/conf.py

```diff
@@ -14,17 +14,18 @@
 #
 import os
 import sys
+import datetime
 sys.path.insert(0, os.path.abspath('..'))
 from gilda import __version__ as gilda_version
 
 # -- Project information -----------------------------------------------------
 
 project = 'gilda'
-copyright = '2020, Benjamin M. Gyori'
+copyright = f'2020-{datetime.datetime.today().year}, Benjamin M. Gyori'
 author = 'Benjamin M. Gyori'
 
 # The short X.Y version
-version = '.'.join(gilda_version.split('.')[:1])
+version = '.'.join(gilda_version.split('.')[1:])
```

Comment:
Both the old implementation and this one are incorrect. It should actually be
```
version = '.'.join(gilda_version.split('.')[:2])
```

Reply:
So with that it will show `0.8` but since we're using a zero versioning schema for gilda, the leading 0 isn't really meaningful